components/root/root.jsx

```diff
@@ -118,15 +122,19 @@ export default class Root extends React.PureComponent {
                 console.log('detected logout from a different tab'); //eslint-disable-line no-console
                 GlobalActions.emitUserLoggedOutEvent('/', false, false);
             }
+            if (e.originalEvent.key === StoragePrefixes.LOGIN && isNewLocalStorageEvent(e)) {
+                const isLoggedIn = getCurrentUser(store.getState());
 
-            if (e.originalEvent.key === StoragePrefixes.LOGIN && e.originalEvent.storageArea === localStorage && e.originalEvent.newValue) {
                 // make sure it isn't this tab that is sending the logout signal (only necessary for IE11)
-                if (BrowserStore.isSignallingLogin(e.originalEvent.newValue)) {
+                if (BrowserStore.isSignallingLogin(e.originalEvent.newValue) || isLoggedIn) {
```

Comment:
Why make this change to look if we have a current user? We might not actually need this if block at all since we've dropped IE11 support unless you needed this on other browsers as well.

Reply:
If I don't check for current user while reloading, one tab will send loginSignal to another, another tab will again send to source tab and all tabs will keep on reloading....
So the check is performed to only reload if the tab is not logged in previously

Reply:
I have removed checking for IE 11 specific constraints in latest changes